install.js

```diff
@@ -1,7 +1,20 @@
 #!/usr/bin/env node
 
-var Validate = require('git-validate');
-Validate.copy('coffeelint.json', 'coffeelint.json');
-Validate.copy('.coffeelintignore', '.coffeelintignore');
-Validate.copy('.codeclimate.yml', '.codeclimate.yml');
-Validate.copy('.editorconfig', '.editorconfig');
+path = require('path');
+fs = require('fs-sync');
```

Comment:
FYI. Most times I'm using `fs-extra` if I need additional methods. But of course, it doesn't matter what you use if it fulfills its purpose.